{"name": "test_login_uninvalid_username[data0]", "status": "passed", "parameters": [{"name": "data", "value": "{'username': '18585436789', 'pwd': '1', 'expected': '此账号没有经过授权，请联系管理员!'}"}], "start": 1612537470963, "stop": 1612537471557, "uuid": "bc82d307-301e-4ec2-b627-3bc281cf52d7", "historyId": "fa5fd66b63c2bcd93bb5f71880ea55e5", "testCaseId": "fc3697ccfb19cd021c9297d4c4fc5f65", "fullName": "tests.test_login.TestLogin#test_login_uninvalid_username", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "J8WB1KTCQOB0HW1"}, {"name": "thread", "value": "13652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}